security/cmd/node_agent_k8s/main.go

```diff
@@ -82,15 +82,15 @@ func init() {
 
 	flags.StringVar(&cAClientConfig.Env, "env", "unspecified",
 		"Node Environment : unspecified | onprem | gcp | aws")
-	flags.StringVar(&cAClientConfig.Platform, "platform", "vm", "The platform istio runs on: vm | k8s")
+	flags.StringVar(&cAClientConfig.Platform, "platform", "k8s", "The platform istio runs on: vm | k8s")
 
 	flags.StringVar(&cAClientConfig.CertChainFile, "cert-chain",
 		"/etc/certs/cert-chain.pem", "Node Agent identity cert file")
 	flags.StringVar(&cAClientConfig.KeyFile,
 		"key", "/etc/certs/key.pem", "Node Agent private key file")
 	flags.StringVar(&cAClientConfig.RootCertFile, "root-cert",
 		"/etc/certs/root-cert.pem", "Root Certificate file")
-	flags.StringVar(&naConfig.SecretDirectory, "secret-dir", "/etc/certs/workload", "The default directory for file based SecretServer")
+	flags.StringVar(&naConfig.SecretDirectory, "secret-dir", "/tmp/workload", "The default directory for file based SecretServer")
```

Comment:
/var/lib/istio/certs maybe - that's where VMs store them.

Reply:
Thanks for the review, WIP. It's not for 0.8(not OSS, we may move the code later); sg /tmp isn't appropriate to store proxies cert key, will sync offline for a better place. + @wattli